internal/repositories/accountMongoRepository.go

```diff
@@ -0,0 +1,149 @@
+package repositories
+
+import (
+	"context"
+	"crud-golang/internal/models"
+	"crud-golang/pkg/client/mongodb"
+	"crud-golang/pkg/logging"
+	"errors"
+	"go.mongodb.org/mongo-driver/bson"
+	"go.mongodb.org/mongo-driver/bson/primitive"
+	"go.mongodb.org/mongo-driver/mongo"
+	"time"
+)
+
+var accountsCollections *mongo.Collection = mongodb.GetCollection(mongodb.DB, "accounts")
+
+type repository struct {
+	accountsCollections *mongo.Collection
+}
+
+func (r repository) Create(user *models.Account) (models.Account, error) {
```

Comment:
It would be best if you pass the context as the first parameter. If you want to create a timeout context, just use the passed one as the parent, instead of context.Background. This way, you really can't really react on canceled request.